server/services/cli.ts

```diff
@@ -1 +0,0 @@
-export const root = process.env.ROOT || process.cwd();
```

Comment:
I don't think we want to delete this :)

Reply:
Nope, messing it up again, I wanted to delee my cli as I was using a custom string sorry too late to PR I guess